pyproject.toml

```diff
@@ -76,7 +76,10 @@ filelock = "^3.8.0"
 checksumdir = "^1.2.0"
 
 [tool.poetry.extras]
-pydantic = ["pydantic", "email-validator"]
+parser = ["pydantic", "email-validator"]
+validation = ["fastjsonschema"]
+tracer = ["aws-xray-sdk"]
+all = ["pydantic", "email-validator", "aws-xray-sdk", "fastjsonschema"]
 
```

Comment:
we need to take into account `jmespath` too. 

`jmespath = ["jmespath"]` 

Every utility providing an `envelope` uses `jmespath` under the hood... which came with `boto3` or `aws-xray-sdk`. For example:  `parser`, `idempotency`, etc.

`idempotency = ["jmespath"]`